src/medCore/views/medAbstractLayeredView.cpp

```diff
@@ -309,7 +309,6 @@ void medAbstractLayeredView::insertLayer(unsigned int layer, medAbstractData *da
 
     if ( this->contains(data))
     {
-        qDebug() << "Attempt to add twice the same data to the view: " << this;
```

Comment:
Removing this one bothers me because it can signify a programming error. I would keep just this message.